src/webnn_native/dml/GraphDML.cpp

```diff
@@ -490,6 +489,34 @@ namespace webnn_native { namespace dml {
         return {};
     }
 
+    ::dml::FusedActivation CreateFusedActivation(OperatorBase* activation) {
+        ::dml::FusedActivation dmlActivation;
+        if (activation != nullptr) {
+            switch (activation->GetOperatorType()) {
+                case OperatorType::Clamp: {
+                    dawn::WarningLog() << "Clamp operator is not supported for fusion on DML.";
```

Comment:
This should not be a warning. If a backend won't support a fused activation, it should just add it as a standalone node into the graph.

Reply:
OK, I will remove it. Thanks!